RELEASE.md

```diff
@@ -0,0 +1,25 @@
+Release type: minor
+
+This release adds support for printing default values for scalars like JSON.
+
+For example the following:
+
+```python
+import strawberry
+from strawberry.scalars import JSON
+
+
+@strawberry.input
+class MyInput:
+    j: JSON = strawberry.field(default_factory=dict)
+    j2: JSON = strawberry.field(default_factory=lambda: {"hello": "world"})
+```
```

Comment:
Does `strawberry.field(default=...)` already work? The description says "default values" but the implementation here seems to be only for "default factories"?

Reply:
yes, it works, but you shouldn't use it for mutable values 😊

Reply:
Ahh right. Of course